src/vl-rich-data-table.js

```diff
@@ -197,6 +223,20 @@ export class VlRichDataTable extends VlElement(HTMLElement) {
         this.__observeSearchFilter();
     }
 
+    /**
+     * Verberg de filter search kolom. Enkel relevant als het filter slot wordt gebruikt.
+     */
+    hideFilterColumn() {
+        this.__setGridColumnWidth(0);
+    }
+
+    /**
+     * Toon de filter search kolom. Enkel relevant als het filter slot wordt gebruikt.
+     */
+    showFilterColumn() {
+        this.__setGridColumnWidth(VlRichDataTable._defaultSearchColumnSize);
+    }
```

Comment:
Misschien best ook rekening houden met a11y en het aria-hidden attribuut zetten zodat ook slechtzienden/blinden de filter kunnen sluiten.

Reply:
Ik zet ze al op `hidden`, en hier lees ik dat dat dan niet meer moet: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute

Reply:
Je verwart hier `hidden`en `aria-hidden` denk ik?

Reply:
Bij deze 2 functies zet je toch geen `hidden` attribuut of kijk ik er naast?

Reply:
> Bij deze 2 functies zet je toch geen `hidden` attribuut of kijk ik er naast?

Ik keek er dus naast, maar dan zou ik de benaming van `__setGridColumnWidth` wel wijzigen, want het doet meer dan de breedte aanpassen?

Reply:
Ok ik zal het wat verduidelijken